internal/idp/resource_manager.go

```
@@ -69,263 +68,111 @@ func (b *ResourceManagerBuilder) Build() (result *ResourceManager, err error) {
 	if err != nil {
 		// Clean up viewers group on failure
-		viewersGroupID, getErr := m.getGroupIDByPath(ctx, organizationName, viewersGroupPath)
+		viewersGroupID, getErr := m.getGroupIDByPath(ctx, tenant, viewersGroupPath)
 		if getErr != nil {
 			m.logger.ErrorContext(ctx, "Failed to get viewers group ID during rollback",
 				slog.String("group_path", viewersGroupPath),
 				slog.Any("get_error", getErr),
 			)
-			// Return composite error including lookup failure
-			return fmt.Errorf("failed to create managers group: %w (rollback also failed to lookup viewers group: %w)", err, getErr)
+			return fmt.Errorf("failed to create managers group: %w (rollback also failed to lookup viewers group: %v)", err, getErr)
```

golangci-lint (2.12.2)

[error] non-wrapping format verb for fmt.Errorf. Use `%w` to format errors

(errorlint)